internal/pkg/cli/job_init.go

```diff
@@ -6,31 +6,72 @@ package cli
 import (
 	"errors"
 	"fmt"
+	"strings"
 
 	"github.com/aws/copilot-cli/internal/pkg/aws/sessions"
 	"github.com/aws/copilot-cli/internal/pkg/cli/group"
 	"github.com/aws/copilot-cli/internal/pkg/config"
 	"github.com/aws/copilot-cli/internal/pkg/deploy/cloudformation"
+	"github.com/aws/copilot-cli/internal/pkg/manifest"
 	"github.com/aws/copilot-cli/internal/pkg/term/color"
 	"github.com/aws/copilot-cli/internal/pkg/term/log"
 	termprogress "github.com/aws/copilot-cli/internal/pkg/term/progress"
+	"github.com/aws/copilot-cli/internal/pkg/term/prompt"
 	"github.com/aws/copilot-cli/internal/pkg/workspace"
 	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
 )
 
-// var (
-// 	fmtJobInitNameHelpPrompt = `The name will uniquely identify this job within your app %s.
-// Deployed resources (such as your job, logs) will contain this job's name and be tagged with it.`
+var (
+	jobInitSchedulePrompt = "How would you like to " + color.Emphasize("schedule") + " this job?"
+	jobInitScheduleHelp   = `How to determine this job's schedule. "Rate" lets you define the time between 
+executions and is good for jobs which need to run frequently. "Fixed Schedule"
+lets you use a predefined or custom cron schedule and is good for less-frequent 
+jobs or those which require specific execution schedules.`
 
-// 	jobInitDockerfileHelpPrompt = "Dockerfile to use for building your job's container image."
-// )
+	jobInitRatePrompt = "How long would you like to wait between executions?"
+	jobInitRateHelp   = `You can specify the time as a duration string. (For example, 2m, 1h30m, 24h)`
+
+	jobInitCronSchedulePrompt = "What schedule would you like to use?"
+	jobInitCronScheduleHelp   = `Predefined schedules run at midnight or the top of the hour.
+For example, "Daily" runs at midnight. "Weekly" runs at midnight on Mondays.`
+	jobInitCronCustomSchedulePrompt = "What custom cron schedule would you like to use?"
+	jobInitCronCustomScheduleHelp   = `Custom schedules can be defined using the following cron:
+Minute | Hour | Day of Month | Month | Day of Week
+For example: 0 17 ? * MON-FRI (5 pm on weekdays)
+			 0 0 1 */3 * (on the first of the month, quarterly)`
+)
```

Comment:
Wow-- you managed to explain something super complicated very clearly and succinctly; I like those examples, too.

Reply:
+1 very good example! Should we make those two examples aligned with each other?

Reply:
Would it be unCopiloty to throw in a link to a reference, in case cron is totally new to a user?